spec/prelude/main.md

```diff
@@ -53,17 +53,35 @@ macro_rules! throw_memory_leak {
         do yeet TerminationInfo::MemoryLeak
     };
 }
+
 macro_rules! throw_ill_formed {
-    () => {
-        do yeet TerminationInfo::IllFormed
+    ($($tt:tt)*) => {
+        do yeet TerminationInfo::IllFormed(format!(
+            "{} at {}:{}:{}",
+            format!($($tt)*),
+            file!(),
+            line!(),
+            column!()
```

Comment:
I don't think I am a fan of this -- and it's not super useful either as it will point to the specr-transpile produced code, not the original sources.

Please make this behave just like `throw_ub`.